pkgs/build-support/node/fetch-npm-deps/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib, stdenv, rustPlatform, Security }:
+
+rec {
+  fetch-npm-deps = rustPlatform.buildRustPackage {
+    pname = "fetch-npm-deps";
+    version = (lib.importTOML ./Cargo.toml).package.version;
+
+    src = ./.;
+
+    cargoLock.lockFile = ./Cargo.lock;
+
+    buildInputs = lib.optional stdenv.isDarwin Security;
+  };
+
+  fetchNpmDeps =
+    { name ? "npm-deps"
+    , src ? null
+    , srcs ? [ ]
+    , patches ? [ ]
+    , sourceRoot ? [ ]
+    , hash ? ""
```

Comment:
Should support for `sha256` be added, to match other fetchers? (As well as adding `npmDepsSha256` to `buildNpmPackage`.)

Reply:
Yes, i think that's a good idea.